despf.sh

```diff
@@ -9,7 +9,10 @@ done
 
 domain=${1:-'orig.spf-tools.ml'}
 
+dig +short +norecurse gnu.org @198.41.0.4 || export NO_STRAIGHT_DNS=1
```

Comment:
This test doesn't print any value, but also returns success.
``
./despf.sh spf-orig.peerj.com
dig: couldn't get address for ';;': not found
```

`dig +short gnu.org NS` returns an error